website/content/docs/job-specification/proxy.mdx

```diff
@@ -60,9 +60,8 @@ job "countdash" {
 - `expose` <code>([expose]: nil)</code> - Used to configure expose path configuration for Envoy.
   See Consul's [Expose Paths Configuration Reference](https://developer.hashicorp.com/consul/docs/connect/registration/service-registration#expose-paths-configuration-reference)
   for more information.
-- `config` `(map: nil)` - Proxy configuration that's opaque to Nomad and
-  passed directly to Consul. See [Consul Connect's
-  documentation](https://developer.hashicorp.com/consul/docs/connect/proxies/envoy#dynamic-configuration)
+- `config` `(map: nil)` - Proxy configuration that is opaque to Nomad and
```

Comment:
Oh was this already documented, but not working as claimed?

Reply:
This is the doc string for `proxy.config` which is basically the same thing but for a different field. I tweaked the wording on this one to eliminate apostrophes